openverse_api/catalog/settings.py

```diff
@@ -37,6 +37,8 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DJANGO_DEBUG_ENABLED", default=False, cast=bool)
 
+PYTHON_ENV = config("PYTHON_ENV", default="development")
```

Comment:
I think we use [`dev`](https://github.com/Automattic/openverse-infrastructure/blob/main/environments/dev/main.tf#L62) and [`prod`](https://github.com/Automattic/openverse-infrastructure/blob/main/environments/prod/main.tf#L65) elsewhere, should we keep this consistent?

Reply:
Ah, sure. We use the full names in the frontend when using it to change runtime behavior... I wonder if that's even working if we're using abbreviations in the terraform variables.

I actually don't think they're even related, cause `NODE_ENV` isn't being set anywhere in terraform as far as I can tell.

Reply:
Also, the default here should be `production`/`prod` whatever string we end up using, to be consistent with `DEBUG`. I think the best is explicitly to indicate the environment in the `.env` file.

Reply:
@krysal Agreed to align the default with `DEBUG`. If it's best to define it explicitly though, should we exclude the default altogether?

Reply:
It could be without a default, I don't have a strong preference here. Django defaults point to production. Our default in the `env.example` can be `development` for quicks setups.

Reply:
Okay, yeah I think it makes sense to have the `env.example` be dev focused with the defaults be production focused.